src/gateways/MsSQL/retrieveActiveDepartmentsByOrganisationId.js

```diff
@@ -0,0 +1,11 @@
+import mssql from "mssql";
+export default ({ getMsSqlConnPool }) => async (organisationId) => {
+  const db = await getMsSqlConnPool();
+  const res = await db
+    .request()
+    .input("organisation_id", mssql.Int, organisationId)
+    .query(
+      `SELECT dbo.[facility].name AS hospitalName, dbo.[department].name AS wardName, dbo.[department].code AS wardCode, dbo.[department].id AS wardId FROM dbo.[facility] JOIN dbo.[department] ON dbo.[facility].id = dbo.[department].facility_id  WHERE dbo.[facility].organisation_id = @organisation_id AND dbo.[department].status = 1 ORDER BY dbo.[department].name ASC;`
```

Comment:
It'd be best to pass the status in here via `.input()` just to make it clearer

Reply:
Done